tests/conftest.py

```diff
@@ -315,15 +316,21 @@ def logged_in_account():
     ]
 }
 default_fixtures[TestLevels.MINIMAL] = {
-    "cluster": ["ondemand_cpu_cluster", "kubernetes_cpu_cluster"]
+    "cluster": [
+        "ondemand_aws_cluster",
+        "ondemand_gcp_cluster",
+        "ondemand_aws_https_cluster_with_auth",
+        "kubernetes_cpu_cluster",
```

Comment:
should we add kubernetes to thorough/maximal too? and rename it to something like `ondemand_k8_cluster` for consistency